plugin/template/project/{{ plugin_name }}/.gitignore

```
@@ -0,0 +1,18 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+idea/*
+
```

Ref: https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore

```suggestion
.history
.vscode/*
.vsix

```